test/integration/test_visual_recognition_v4.py

```diff
@@ -128,40 +127,38 @@ def test_04_objects_and_training(self):
 
         # list objects metadata
         object_metadata_list = self.visual_recognition.list_object_metadata(
-            collection_id=self.collection_id).get_result()
+            collection_id=collection_id).get_result()
         assert object_metadata_list is not None
 
         # update object metadata
         object_metadata = object_metadata_list.get('objects')[0]
         updated_object_metadata = self.visual_recognition.update_object_metadata(
-            collection_id=self.collection_id,
+            collection_id=collection_id,
             object=object_metadata.get('object'),
             new_object='updated giraffe training data').get_result()
         assert updated_object_metadata is not None
 
         # get object metadata
         object_metadata = self.visual_recognition.get_object_metadata(
-            collection_id=self.collection_id,
+            collection_id=collection_id,
             object='updated giraffe training data',
         ).get_result()
         assert object_metadata is not None
         assert object_metadata.get('object') == 'updated giraffe training data'
 
         # train collection
-        train_result = self.visual_recognition.train(self.collection_id).get_result()
+        train_result = self.visual_recognition.train(collection_id).get_result()
         assert train_result is not None
         assert train_result.get('training_status') is not None
 
         # training usage
-        # training_usage = self.visual_recognition.get_training_usage(
-        #     start_time='2019-11-01', end_time='2019-11-27').get_result()
-        # assert training_usage is not None
+        training_usage = self.visual_recognition.get_training_usage(
+            start_time='2019-11-01', end_time='2019-11-27').get_result()
+        assert training_usage is not None
```

Comment:
Does this pass? its looking only in nov 2019

Reply:
haha, it's still passing somehow. Good catch. Probably cause we are only asserting `assert training_usage is not None`.